mandatory/4-tax.js

```diff
@@ -16,9 +18,9 @@ function calculateSalesTax() {}
 
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
-
-function addTaxAndFormatCurrency() {}
-
+function addTaxAndFormatCurrency(number) {
+  return "£" + calculateSalesTax(number).toFixed(2);
+}
 /* 
```

Comment:
Great job on these functions, Can you please have a look at the other function ??